sdk/documenttranslation/azure-ai-documenttranslation/samples/async_samples/sample_batch_translation_with_storage_async.py

```diff
@@ -0,0 +1,141 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import os
+import asyncio
+
+
+class BatchTranslationWithStorageSampleAsync(object):
+
+    async def batch_translation_with_storage_async(self):
+        # import libraries
+        from azure.core.credentials import AzureKeyCredential
+        from azure.ai.textanalytics.aio import DocumentTranslationClient
+        from azure.ai.documenttranslation import (
+            BatchDocumentInput,
+            StorageTarget
+        )
+        from azure.storage.blob import ContainerClient, generate_container_sas, ContainerSasPermissions
```

Comment:
we should import ContainerClient from the `aio` namespace

Reply:
so you would like to have the storage methods be async also?